src/main/java/com/example/parser/utils/FileDownload.java

```diff
@@ -0,0 +1,51 @@
+package com.example.parser.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class FileDownload {
+
+    private static File tmpFile = null;
+    private static final Logger log = LoggerFactory.getLogger(FileDownload.class);
+
+    public static File download(String fileUrl) {
+
+        BufferedInputStream in = null;
+        BufferedOutputStream bout = null;
+        byte[] buffer = new byte[1024];
+
+        try {
+            tmpFile = File.createTempFile("dozvoli-na-vikidi-onovlenii", ".tmp");
```

Comment:
Do not set a temporary file name manually. It will create a problem when few files will be downloaded simultaneously. Lets OS and Java set name.

Reply:
done